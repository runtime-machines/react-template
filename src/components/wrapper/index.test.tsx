import * as React from 'react';
import { shallow } from 'enzyme';

import { Wrapper } from '.';

describe('Wrapper component', () => {
	it('should render successfully', () => {
		const tree = shallow(<Wrapper>Test content</Wrapper>);

		expect(tree).toMatchSnapshot();
	});
});
