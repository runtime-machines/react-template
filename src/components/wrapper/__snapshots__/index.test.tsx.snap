// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper component should render successfully 1`] = `
<div
  className="o-wrapper"
>
  <Header />
  <main
    className="o-main"
  >
    Test content
  </main>
  <Footer />
</div>
`;
